### PR TITLE
Fixed bugs and added tests

### PR DESCRIPTION
**Pull request by germanbisurgi**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Is backward-compatible?    | ✔️
| Tests pass?   | ✔️
| Fixed issues  | #941, #919, #906, #873, #797, #505, #807, #917, #909, #755, #509, #130, #926, #877, #749, #946
| Updated README/docs?   | ✔️
| Added CHANGELOG entry?   | ❌

### CODE REVIEW

File: tests/codeceptjs/editors/array_test.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ Scenario('should trigger array (table) editing triggers', async (I) => {
 
   I.amAcceptingPopups();
   I.click('//button[contains(@class, "json-editor-btn-moveup") and @data-i="1"]');
+  I.wait(1)
```
Comment (schmunk42) [inline tests/codeceptjs/editors/array_test.js:24]: is there no `IwaitForPopup` or `...Element` or similar?

Comment (germanbisurgi) [reply tests/codeceptjs/editors/array_test.js:24]: there is no waitForPopUp yet unfortunately

Comment (schmunk42) [reply tests/codeceptjs/editors/array_test.js:24]: Would it be possible to write a custom webdriver helper? Similar to https://stackoverflow.com/a/31453019

**Comment by robocoder** [comment]
Don't forget to merge the release branch to master.

**Comment by schmunk42** [comment]
> Don't forget to merge the release branch to master.

We actually only merge on release from master, just for releases.
Not super-nice, but it's the current workflow.

**Comment by schmunk42** [comment]
@germanbisurgi Did you also try https://codecept.io/basics/#retry-step ?

**Comment by germanbisurgi** [comment]
@schmunk42: retry-step made the trick. I think this PR is ready to merge.

**Review by schmunk42 — APPROVED** [review]

**Pull request merged**